runtime/server/x86/README.md

```diff
@@ -139,19 +139,33 @@ wav_path=your_test_wav_path
 ## Run the demo in Docker
 
 When you encounter an issue trying to run the demo, we encourage you to run the demo in
-the Docker container. The image contains the latest release, a shell script and
-several waves to run the demo. Just run it as follows:
+the Docker container.
+
+Or build the Dockerfile yourself, and run it by:
```

Comment:
You can use our prebuilt docker, (demo)

Or you can build your own docker by (demo)